src/Tests/test-data/load-balancer.json

```diff
@@ -26,7 +26,8 @@
                     "serviceName": "Microsoft.ContainerInstance/containerGroups"
                   }
                 }
-              ]
+              ],
+              "privateEndpointNetworkPolicies": "Enabled"
```

Comment:
I've modified the builder and ARM model to now take a `FeatureFlag option` so that if no value is set (and the value is therefore `None`) we will emit `null` into the JsonModel so it won't be serialized into the model